### ISSUE TITLE
Newspack Popups are not editable anymore

### ISSUE THREAD

**Issue by estereofonico** [Automattic/newspack-popups#571]
Since the last two releases i've been experiencing this error: /wp-content/plugins/newspack-plugin/includes/configuration_managers/class-newspack-popups-configuration-manager.php on line 97

In the Admin Screen I can't display the active campaigns or edit the campaign. Could you please help me to solve this issue? The only error available is a screen that only says "Unrecoverable error". 

I guess the option was not created or updated in recent versions and now is asking for something that the service can't create or update. 

Thanks for your help.

**Comment by dkoo** [comment]
Hi @estereofonico, did you make sure your site is also running the latest version of the [Newspack Plugin](https://github.com/Automattic/newspack-plugin/releases/tag/v1.48.1)? It would also help if you could paste the entire error message (including stack trace) from the JS console.

**Comment by estereofonico** [comment]
Yes, the error isn't new for me. Since v1.22 i've disabled this feature to avoid errors. I decided to ask for your help because the issue hasn't been solved in the last version. Our main site currently use v1.43 but i've post this error after trying with v.1.48.0 that also give us a big error trying to find the main page template, the ads and the campaigns... error in the other plugins, not the main one.

Let me try again with the new release and I let you know what happens.

**Comment by dkoo** [comment]
If you're able to provide some screenshots of the JS console errors you see when that "unrecoverable error" happens, that might be helpful. And/or the response code from the Network dev tools tab from any failing API requests, if possible. The more info we have, the better we can pinpoint where the issue is occurring.

In the meantime, please do try v1.48.1 of the primary plugin along with v1.35.0 of Campaigns and let us know if that solves the issue for you.

**Comment by estereofonico** [comment]
We already did your suggestion but wait until 1.49.0 and the error is:

Fatal error: Uncaught Error: Class 'Newspack_Popups_Custom_Placements' not
found in
/wp-content/plugins/newspack-plugin/includes/configuration_managers/class-newspack-popups-configuration-manager.php:97
Stack trace: #0
/wp-content/plugins/newspack-plugin/includes/wizards/class-popups-wizard.php(475):
Newspack\Newspack_Popups_Configuration_Manager->get_custom_placements() #1
/wp-includes/class-wp-hook.php(303):
Newspack\Popups_Wizard->enqueue_scripts_and_styles('newspack_page_n...') #2
/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters(NULL, Array) #3
/wp-includes/plugin.php(470): WP_Hook->do_action(Array) #4
/wp-admin/admin-header.php(102): do_action('admin_enqueue_s...',
'newspack_page_n...') #5 /wp-admin/admin.php(239): require_once('/...') #6
in
/wp-content/plugins/newspack-plugin/includes/configuration_managers/class-newspack-popups-configuration-manager.php
on line 97

We're still unable to edit our published campaigns or create a new one.

Thanks for your help on this issue.

*Mauricio Cubaque *
*SEO / Marketing Digital *
Twitter: @estereofonico <https://twitter.com/estereofonico> |
Bogotá, Colombia
https://estereofonica.com | https://quintopiso.net |
https://glutationplatino.com

Winners keep track of Results. Loosers keep track of reasons.

**Comment by dkoo** [comment]
@estereofonico Looks like you need to install and activate the latest version of Newspack Campaigns: https://github.com/Automattic/newspack-popups/releases/latest/

**Comment by estereofonico** [comment]
Thanks for your advice, I've installed this version but when trying to
update some features, they don't update when I change settings on the
prompt sidebar like color or frequency.

Regards.

**Comment by dkoo** [comment]
Does this still happen if you edit the prompt in a different browser? We've sometimes seen instances where editor JS is cached and needs a cache clear/hard refresh in order to get updates.

**Comment by adekbadek** [comment]
Closing due to lack of response. 
Please ensure the `newspack-plugin` & `newspack-popups` are at their latest versions when used together.

**Comment by estereofonico** [comment]
Yes, it does. I've tried with Opera and Chrome and the behaviour remains
the same. Thanks for your help. This plugin has a heavy use of resources in
my vps, so I've disabled it and changed to Gutenberg blocks.
